config/initializers/content_deposit_event_job_prepends.rb

```diff
@@ -3,6 +3,7 @@ def action
     super
     if repo_object.deduplication_key
       pre_ingest_work = Zizia::PreIngestWork.find_by(deduplication_key: repo_object.deduplication_key)
+      return nil unless pre_ingest_work
```

Comment:
So, the original class is in Hyrax and looks like this - 

```
# Log a concern deposit to activity streams
class ContentDepositEventJob < ContentEventJob
  def action
    "User #{link_to_profile depositor} has deposited #{link_to repo_object.title.first, polymorphic_path(repo_object)}"
  end
end
```

My POV is that the status stuff is pretty fundamentally broken / unimplemented, so it would make more sense to do a more comprehensive refactor of that on a separate PR, and for now just make sure that it's not blocking other work.